pkg/skaffold/deploy/deploy_mux.go

```diff
@@ -103,5 +78,5 @@ func (m DeployerMux) Render(ctx context.Context, w io.Writer, as []build.Artifac
 	}
 
 	allResources := strings.Join(resources, "\n---\n")
-	return outputRenderedManifests(allResources, filepath, w)
+	return manifest.OutputRenderedManifests(allResources, filepath, w)
```

Comment:
`OutputRenderedManifests` to probably `OutputRender`

Reply:
WDYT about `manifest.Write()`? the fact that the manifests are rendered seems like an implementation detail to me